processing file asmFiles/test.beq_ntaken.asm (1 of 24)
processing file asmFiles/test.beq_taken.asm (2 of 24)
processing file asmFiles/test.bge_ntaken.asm (3 of 24)
processing file asmFiles/test.bge_taken.asm (4 of 24)
processing file asmFiles/test.bgeu_ntaken.asm (5 of 24)
processing file asmFiles/test.bgeu_taken.asm (6 of 24)
processing file asmFiles/test.blt_ntaken.asm (7 of 24)
processing file asmFiles/test.blt_taken.asm (8 of 24)
processing file asmFiles/test.bltu_ntaken.asm (9 of 24)
processing file asmFiles/test.bltu_taken.asm (10 of 24)
processing file asmFiles/test.bne_ntaken.asm (11 of 24)
processing file asmFiles/test.bne_taken.asm (12 of 24)
processing file asmFiles/test.branchjump1.asm (13 of 24)
--- memsim.hex	2024-09-24 14:58:02.747672330 -0400
+++ memcpu.hex	2024-09-24 14:58:02.730671685 -0400
@@ -16,6 +16,5 @@
 :04000F00004526235F
 :04001000FEB094E3C7
 :04001100FFFFFFFFEF
-:0400220000000024B6
-:040024000000003C9C
+:040020000000BA5CC6
 :00000001FF
processing file asmFiles/test.cfhazard_beq.asm (14 of 24)
processing file asmFiles/test.dhazard_add.asm (15 of 24)
processing file asmFiles/test.dhazard_lw.asm (16 of 24)
processing file asmFiles/test.halt.asm (17 of 24)
processing file asmFiles/test.jal.asm (18 of 24)
processing file asmFiles/test.jalr.asm (19 of 24)
processing file asmFiles/test.ldst1.asm (20 of 24)
processing file asmFiles/test.ldst2.asm (21 of 24)
processing file asmFiles/test.loadstore.asm (22 of 24)
processing file asmFiles/test.rtype.asm (23 of 24)
processing file asmFiles/test.shazard_lw.asm (24 of 24)
processing file asmFiles/test.beq_ntaken.asm (1 of 24)
processing file asmFiles/test.beq_taken.asm (2 of 24)
processing file asmFiles/test.bge_ntaken.asm (3 of 24)
processing file asmFiles/test.bge_taken.asm (4 of 24)
processing file asmFiles/test.bgeu_ntaken.asm (5 of 24)
processing file asmFiles/test.bgeu_taken.asm (6 of 24)
processing file asmFiles/test.blt_ntaken.asm (7 of 24)
processing file asmFiles/test.blt_taken.asm (8 of 24)
processing file asmFiles/test.bltu_ntaken.asm (9 of 24)
processing file asmFiles/test.bltu_taken.asm (10 of 24)
processing file asmFiles/test.bne_ntaken.asm (11 of 24)
processing file asmFiles/test.bne_taken.asm (12 of 24)
processing file asmFiles/test.branchjump1.asm (13 of 24)
--- memsim.hex	2024-09-24 15:12:17.591126330 -0400
+++ memcpu.hex	2024-09-24 15:12:17.575125723 -0400
@@ -16,6 +16,8 @@
 :04000F00004526235F
 :04001000FEB094E3C7
 :04001100FFFFFFFFEF
+:040020000000BA5CC6
+:040021000000BA5CC5
 :0400220000000024B6
-:040024000000003C9C
+:040023000000BA5CC3
 :00000001FF
processing file asmFiles/test.cfhazard_beq.asm (14 of 24)
processing file asmFiles/test.dhazard_add.asm (15 of 24)
processing file asmFiles/test.dhazard_lw.asm (16 of 24)
processing file asmFiles/test.halt.asm (17 of 24)
processing file asmFiles/test.jal.asm (18 of 24)
processing file asmFiles/test.jalr.asm (19 of 24)
processing file asmFiles/test.ldst1.asm (20 of 24)
processing file asmFiles/test.ldst2.asm (21 of 24)
processing file asmFiles/test.loadstore.asm (22 of 24)
processing file asmFiles/test.rtype.asm (23 of 24)
processing file asmFiles/test.shazard_lw.asm (24 of 24)
processing file asmFiles/test.beq_ntaken.asm (1 of 24)
processing file asmFiles/test.beq_taken.asm (2 of 24)
processing file asmFiles/test.bge_ntaken.asm (3 of 24)
processing file asmFiles/test.bge_taken.asm (4 of 24)
processing file asmFiles/test.bgeu_ntaken.asm (5 of 24)
processing file asmFiles/test.bgeu_taken.asm (6 of 24)
processing file asmFiles/test.blt_ntaken.asm (7 of 24)
processing file asmFiles/test.blt_taken.asm (8 of 24)
processing file asmFiles/test.bltu_ntaken.asm (9 of 24)
processing file asmFiles/test.bltu_taken.asm (10 of 24)
processing file asmFiles/test.bne_ntaken.asm (11 of 24)
processing file asmFiles/test.bne_taken.asm (12 of 24)
processing file asmFiles/test.branchjump1.asm (13 of 24)
processing file asmFiles/test.cfhazard_beq.asm (14 of 24)
processing file asmFiles/test.dhazard_add.asm (15 of 24)
processing file asmFiles/test.dhazard_lw.asm (16 of 24)
processing file asmFiles/test.halt.asm (17 of 24)
processing file asmFiles/test.jal.asm (18 of 24)
processing file asmFiles/test.jalr.asm (19 of 24)
processing file asmFiles/test.ldst1.asm (20 of 24)
processing file asmFiles/test.ldst2.asm (21 of 24)
processing file asmFiles/test.loadstore.asm (22 of 24)
processing file asmFiles/test.rtype.asm (23 of 24)
processing file asmFiles/test.shazard_lw.asm (24 of 24)
processing file asmFiles/count_days.asm (1 of 30)
--- memsim.hex	2024-09-24 15:29:56.625340791 -0400
+++ memcpu.hex	2024-09-24 15:29:56.607340108 -0400
@@ -1,4 +1,4 @@
-:04000000FFC0011329
+:0400000000002238A2
 :0400010001B00293B5
 :040002000080031364
 :040003007E80039365
@@ -36,6 +36,7 @@
 :04002300FFC1011305
 :0400240000A12023F4
 :0400250000008067F0
-:043FFD0000000018A8
-:043FFE000000223865
+:043FFD0000000007B9
+:043FFE0000000018A7
+:043FFF000000016D50
 :00000001FF
processing file asmFiles/fib.asm (2 of 30)
--- memsim.hex	2024-09-24 15:29:58.221401379 -0400
+++ memcpu.hex	2024-09-24 15:29:58.204400734 -0400
@@ -14,27 +14,27 @@
 :04000D00FE0814E3F2
 :04000E00FFFFFFFFF2
 :0400210000000001DA
-:0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400220000000017C3
+:0400230000000018C1
+:040024000000002FA9
+:040025000000004790
+:040026000000007660
+:04002700000000BD18
+:0400280000000133A0
+:04002900000001F0E2
+:04002A0000000323AC
+:04002B0000000513B9
+:04002C000000083692
+:04002D0000000D4979
+:04002E000000157F3A
+:04002F00000022C8E3
+:04003000000038474D
+:0400310000005B0F61
+:0400320000009356E1
+:040033000000EE6576
+:04003400000181BB8B
+:040035000002702035
+:040036000003F1DBF7
+:04003700000661FB63
 :0403C0000000001623
 :00000001FF
processing file asmFiles/mergesort.asm (3 of 30)
--- memsim.hex	2024-09-24 15:30:03.347595976 -0400
+++ memcpu.hex	2024-09-24 15:30:03.328595255 -0400
@@ -1,4 +1,3 @@
-:04000000FFC06413C6
 :04000100FFC06113C8
 :04000200FFFF71B7D4
 :040003004031013354
@@ -85,61 +84,165 @@
 :040054000047071347
 :0400550000078463B9
 :04005600FE9FF06FAA
-:0400570000008067BE
-:0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
+:0400570000000001A4
+:0400580000000003A1
+:04005900000000049F
+:04005A00000000079B
+:04005B00000000079A
+:04005C000000000799
+:04005D000000000798
+:04005E000000000896
+:04005F000000000895
+:040060000000000894
+:040061000000000992
+:040062000000000A90
+:040063000000000B8E
+:040064000000000C8C
+:040065000000000F88
+:040066000000001086
+:040067000000001184
+:04006800000000157F
+:04006900000000167D
+:04006A00000000167C
+:04006B00000000177A
+:04006C000000001977
+:04006D000000001A75
+:04006E000000001B73
+:04006F000000001D70
+:04007000000000206C
+:04007100000000206B
+:04007200000000206A
+:040073000000002168
+:040074000000002365
+:040075000000002463
+:040076000000002660
+:04007700000000265F
+:04007800000000285C
+:04007900000000295A
+:04007A000000002F53
+:04007B00000000334E
+:04007C00000000364A
+:04007D000000003847
+:04007E000000003D41
+:04007F000000003D40
+:040080000000003E3E
+:040081000000004239
+:040082000000004733
+:040083000000004B2E
+:040084000000004D2B
+:040085000000005027
+:040086000000005125
+:040087000000005124
+:040088000000005321
+:04008900000080678C
+:04008A000000005A18
+:04008B000000005A17
+:04008C000000005A16
+:04008D000000005A15
+:04008E000000005A14
+:04008F000000005A13
+:040090000000005A12
+:040091000000005A11
+:040092000000005A10
+:040093000000005A0F
+:040094000000005A0E
+:040095000000005A0D
+:040096000000005A0C
+:040097000000005A0B
+:040098000000005A0A
+:040099000000005A09
+:04009A000000005A08
+:04009B000000005A07
+:04009C000000005A06
+:04009D000000005A05
+:04009E000000005A04
+:04009F000000005A03
+:0400A0000000005A02
+:0400A1000000005A01
+:0400A2000000005A00
+:0400A3000000005AFF
+:0400A4000000005AFE
+:0400A5000000005AFD
+:0400A6000000005AFC
+:0400A7000000005AFB
+:0400A8000000005AFA
+:0400A9000000005AF9
+:0400AA000000005AF8
+:0400AB000000005AF7
+:0400AC000000005AF6
+:0400AD000000005AF5
+:0400AE000000005AF4
+:0400AF000000005AF3
+:0400B0000000005AF2
+:0400B1000000005AF1
+:0400B2000000005AF0
+:0400B3000000005AEF
+:0400B4000000005AEE
+:0400B5000000005AED
+:0400B6000000005AEC
+:0400B7000000005AEB
+:0400B8000000005AEA
+:0400B9000000005AE9
+:0400BA000000005AE8
+:0400BB000000005AE7
+:0400BC000000005AE6
+:0400BD000000005AE5
+:0400BE000000005AE4
+:0400BF000000005AE3
+:0400C0000000005AE2
+:0400C1000000005AE1
+:0400C2000000005AE0
+:0400C3000000005ADF
+:0400C4000000005ADE
+:0400C5000000005ADD
+:0400C6000000005ADC
+:0400C7000000005ADB
+:0400C8000000005ADA
+:0400C9000000005AD9
+:0400CA000000005AD8
+:0400CB000000005AD7
+:0400CC000000005AD6
+:0400CD000000005AD5
+:0400CE000000005AD4
+:0400CF000000005AD3
+:0400D0000000005AD2
+:0400D1000000005AD1
+:0400D2000000005AD0
+:0400D3000000005ACF
+:0400D4000000005ACE
+:0400D5000000005ACD
+:0400D6000000005ACC
+:0400D7000000005ACB
+:0400D8000000005ACA
+:0400D9000000005AC9
+:0400DA000000005AC8
+:0400DB000000005AC7
+:0400DC000000005AC6
+:0400DD000000005AC5
 :0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
+:0400DF000000005AC3
+:0400E0000000005AC2
+:0400E10000000030EB
+:0400E20000000030EA
+:0400E30000000030E9
+:0400E40000000030E8
+:0400E50000000030E7
+:0400E60000000030E6
+:0400E70000000030E5
+:0400E80000000030E4
+:0400E90000000030E3
+:0400EA0000000030E2
+:0400EB0000000030E1
+:0400EC0000000030E0
+:0400ED0000000030DF
+:0400EE0000000030DE
+:0400EF0000000030DD
+:0400F00000000030DC
+:0400F10000000030DB
+:0400F20000000030DA
+:0400F30000000030D9
+:0400F40000000030D8
+:0400F50000000030D7
 :0400F60000000030D6
 :0400F70000000032D3
 :0400F80000000041C3
@@ -151,69 +254,38 @@
 :0400FE000000005CA2
 :0400FF000000005CA1
 :040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:04014000000000308B
+:040141000000005A60
+:040142000000003089
+:040143000000003088
+:040144000000003087
+:040145000000003086
+:040146000000003085
+:040147000000003084
+:040148000000003083
+:040149000000003082
+:04014A000000003081
+:04014B000000003080
+:04014C00000000307F
+:04014D00000000307E
+:04014E00000000307D
+:04014F00000000307C
+:04015000000000307B
+:04015100000000307A
+:040152000000003079
+:040153000000003078
+:040154000000003077
+:040155000000003076
+:040156000000003075
+:040157000000003272
+:040158000000004162
+:040159000000004260
+:04015A00000000435E
+:04015B00000000445C
+:04015C000000005946
+:04015D000000005B43
+:04015E000000005C41
+:04015F000000005C40
+:040160000000006437
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (4 of 30)
--- memsim.hex	2024-09-24 15:30:04.979657931 -0400
+++ memcpu.hex	2024-09-24 15:30:04.962657286 -0400
@@ -20,5 +20,6 @@
 :0400130000A1202305
 :04001400FFFFFFFFEC
 :043FFD0000000005BB
-:043FFE0000000014AB
+:043FFE0000000004BB
+:043FFF0000000014AA
 :00000001FF
processing file asmFiles/mult_procedure.asm (5 of 30)
